pyocd/flash/loader.py

```diff
@@ -29,6 +29,11 @@
 
 LOG = logging.getLogger(__name__)
 
+
+class ArgumentError(Exception):
```

Comment:
I would not recommend to re-use argparser introduced ArgumentError name like this because it might causes confusion for users. Propose to inherit argparser.ArgumentError here, or maybe even better to use some other Exception, like `ValueError` or `TypeError` which does not cause conflict with name like this.

Reply:
I'll agree with that! I wasn't sure if `ArgumentError` was exported by argparse.

I prefer using `ValueError` or `TypeError`.

Reply:
I'll change to ValueError.